routes/web.php

```diff
@@ -30,3 +31,6 @@
 Route::get('/login/google/callback', [LoginController::class, 'handleGoogleCallback']);
 
 Route::get('logout', [LoginController::class, 'logout'])->name('logout');
+Route::get('wish-list', [WishListController::class, 'index'])->name('wishList.form')->middleware('auth');
+Route::post('create/wish-list', [WishListController::class, 'create'])->name('create.wishList');
```

Comment:
make the route naming with hierarchy prefixes, like:
wishlist.index
wishlist.create

and so on